src/condition_checker.coffee

```diff
@@ -1,7 +1,12 @@
 presenceMethods = ['present', 'blank']
 
 class FormRenderer.ConditionChecker
-  constructor: (@form_renderer, @condition) ->
+  # ConditionChecker may be called from outside the FormRenderer context,
+  # e.g. from FormBuilder. In that case, pass in @field instead of @form_renderer.
+  # NOTE: For the next major release, we can decouple ConditionChecker from the
+  # @form_renderer object altogether by just forcing all external callers to pass
+  # in @field
+  constructor: (@form_renderer, @condition, @field) ->
```

Comment:
Will this break any existing code that's expecting only two parameters in this constructor?

Reply:
I think (but am not 100% sure) that coffeescript allows variable numbers of args to a method. Are there any exceptions to this?

Reply:
http://coffeescript.org/#try:class%20Test%0A%20%20constructor%3A%20(a%2C%20b%2C%20c)%20-%3E%0A%20%20%20%20console.log%20a%0A%20%20%20%20console.log%20b%0A%20%20%20%20console.log%20c%0A%20%20%20%20%0A%20%0At1%20%3D%20new%20Test%0A%0At2%20%3D%20new%20Test(1)%0A%0At3%20%3D%20new%20Test(1%2C%202)%0A%0At4%20%3D%20new%20Test(1%2C%202%2C%203)

Reply:
That seems to work ok

Reply:
Ah, I had forgotten that was the case in coffeescript. Looks good here then. 👍